src/app/aether-application/application-edit/application-edit.component.ts

```diff
@@ -158,14 +155,16 @@ export class ApplicationEditComponent extends RocEditBase implements OnInit {
 
     deleteFromSelect(ep: string): void {
         this.bs.deleteIndexedEntry(
-            '/application-2.0.0/application[id=' +
+            '/' +
```

Comment:
Not necessarily pertinent to this PR, but in general I think we should move to use template strings: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Will make the code more concise and more readable:
```js
'/' +
this.pathRoot +
'/application[application-id=' +
this.id +
']/endpoint[endpoint-id=' +
ep +
']',
```
will become
```js
`/${this.pathRoot}/application[application-id=${this.id}]/endpoint[endpoint-id=${ep}]`,
```

Reply:
good idea - i might not be able to get to it today - but should be done